### PR TITLE
address issue #278

### PR DESCRIPTION
**Pull request by timoschowski**
*Issue #, if available:*
#278
*Description of changes:*
This adds the version number of gluonts to the type.txt file during serialization, but doesn't do anything with this during deserialization. I tested this locally, but didn't add any unit tests. If I should, please point me to where I'd best do this.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by jaheba** [comment]
Thanks!

I think it would be better to create a new file to store the version into. At the moment you store it into `type.txt` which should just contain the class-path of the predictor.

Maybe we can add a `version.json` which could store two fields:

* the model's version (via `self.__version__`)
* the version of gluon-ts (`gluonts.__version__`)

For gluon-ts based algorithms, these are the same, but external algorithms can have independent ones. Thus, we should store both versions.

File: src/gluonts/model/predictor.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
 from pathlib import Path
 from pydoc import locate
 from tempfile import TemporaryDirectory
+import json
```
Comment (jaheba) [inline src/gluonts/model/predictor.py]: I think normally we would want imports sorted alphabetically, but I don't mind too much.

**Comment by szha** [comment]
Job PR-388/2 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-388/2/index.html

**Comment by szha** [comment]
Job PR-388/3 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-388/3/index.html

**Comment by szha** [comment]
Job PR-388/4 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-388/4/index.html

**Pull request merged**